1-js/05-data-types/02-number/article.md

```diff
@@ -239,22 +239,22 @@ Can we work around the problem? Sure, there's a number of ways:
     alert( sum.toFixed(2) ); // 0.30
     ```
 
-    Please note that `toFixed` always returns a string. It ensures that it has 2 digits after the decimal point. That's actually convenient if we have an e-shopping and need to show `$0.30`. For other cases we can use the unary plus to coerce it into a number:
+    Please note that `toFixed` always returns a string. It ensures that it has 2 digits after the decimal point. That's actually convenient if we have an e-shopping and need to show `$0.30`. For other cases, we can use the unary plus to coerce it into a number:
 
     ```js run
     let sum = 0.1 + 0.2;
     alert( +sum.toFixed(2) ); // 0.3
     ```
 
-2. We can temporarily turn numbers into integers for the maths and then go back. That would looks like this:
+2. We can temporarily turn numbers into integers for the maths and then revert it back. It works like this:
 
     ```js run
     alert( (0.1*10 + 0.2*10) / 10 ); // 0.3
     ```
 
-    It works, because when we get `0.1*10 = 1` and `0.2 * 10 = 2` then both numbers are integers, there's no precision loss for them. 
+    This works because when we do `0.1*10 = 1` and `0.2 * 10 = 2` then both numbers become integers, and there's no precision loss. 
 
-3. If it's a shop, then the most radical solution would be to store all prices in cents. No fractions at all. But what if we apply a discount of 30%? In practice, totally evading fractions is rarely feasible, so the solutions listed above are here to help.
+3. If it's we were dealing with a shop, then the most radical solution would be to store all prices in cents and use no fractions at all. But what if we apply a discount of 30%? In practice, totally evading fractions is rarely feasible, so the solutions above help avoid this pitfall.
```

Comment:
"If it's we were dealing with..." -- can't understand this.

Reply:
Looks like a typo. Maybe @gwooly meant to remove "it's"? That would make sense.

Reply:
Yes correct, removed 'it's'.